src/com/amazon/ion/impl/lite/IonContainerLite.java

```diff
@@ -88,6 +97,12 @@ protected IonContainerLite(ContainerlessContext context, boolean isNull)
         }
     }
 
+    // See the comment on the `ionSystem` member field for more information.
+    @Override
+    public IonSystemLite getSystem() {
+        return ionSystem;
+    }
```

Comment:
Is the override on `IonSystemLite`?  I would've expected `IonSystem`, but I can't see the root of the API

Reply:
I was surprised too! Here's the signature from the `IonContext` interface:

https://github.com/amzn/ion-java/blob/d5939190808b815ec940ef5d45db0d39e9c3fb63/src/com/amazon/ion/impl/lite/IonContext.java#L60

and the same signature in `IonValueLite`:

https://github.com/amzn/ion-java/blob/02169fee3665f062e92823a32d6f57e05f924ac2/src/com/amazon/ion/impl/lite/IonValueLite.java#L647